docs/guides/notebooks/faq/index.rst

```diff
@@ -105,3 +105,9 @@ How to make Git stop asking for my password?
 --------------------------------------------
 
 Follow the :doc:`guide on configuring Git and credentials <../configuration/git-configuration>` to set your Git username and set up credentials for services like GitHub.
+
+
+Can I access shared data for rubin_sim?
+---------------------------------------
```

Comment:
I would move this up so that it's not under "Troubleshooting tips".

Reply:
Good idea, done.